.github/workflows/add-to-bl.yml

```
@@ -0,0 +1,17 @@
+name: Add new issues to the backlog
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/Clinical-Genomics/projects/45
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

It's probably a better idea to use the [automatic token authentication](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) that having an empty secret, what do you think?

```suggestion
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

---

I've added an org PAT now. Should work. It will expire at the end of the year, so if this breaks that's why :)